src/components/paragraph/sul-home-banner/sul-home-banner.client.tsx

```diff
@@ -52,11 +52,15 @@ export const SulHomeBannerFormClient = () => {
         </label>
         <select
           id={`${inputId}-action`}
-          className="h-40 w-full border-0 bg-none text-16 font-semibold hover:cursor-pointer md:w-auto md:min-w-[15rem] md:text-20 xl:text-22"
+          className="h-40 w-full border-0 bg-none text-16 font-semibold leading-normal hover:cursor-pointer md:w-auto md:min-w-[15rem] md:text-20 xl:text-22"
           onChange={e => setFormAction(e.target.value)}
           value={formAction}
         >
-          <option value="/all">All resources</option>
+          <option value="/all">All library resources</option>
+          <option value="/catalog">Catalog</option>
+          <option value="/articles?q=climate+change&f[eds_search_limiters_facet][]=Direct+access+to+full+text">
```

Comment:
just set the value to `https://searchworks.stanford.edu/articles`

Reply:
@pookmish any tips on how we might handle adding the additional queries? SUL would like `&f[eds_search_limiters_facet][]=Direct+access+to+full+text` appended to the search query 🤔

Reply:
You can have a hidden field that is conditionally rendered based on the formAction state:
```
{formAction === "foobar" && <input type="hidden" name="f[eds_search_limiters_facet][]" value="Direct access to full text" />
```

Reply:
Sweet! I'll play around with this. Thanks Mike 🫡